.github/workflows/surge_preview.yml

```diff
@@ -0,0 +1,42 @@
+name: Pull Request Preview
+
+on:
+  workflow_run:
```

Comment:
This only runs on the main branch, so need to merge to actually see it in action